### ISSUE TITLE
Updating rider basics

### ISSUE THREAD

**Issue by abhilash710** [TeleportXYZ/TRIP-Guides#27]
### What's missing from the TRIP Guides
A placeholder for riders to help them understand what they can expect here. This is also a place for drivers to understand the rider experience so that they feel confident that riders have the same UX as what they've known.

### Expected content
- How does using Teleport compare to Uber/Lyft?
- Screenshots of the app
- Lightly touch on the web3 elements 

### Actual content
On TRIP riders [pay a lot less](https://app.gitbook.com/o/0WSd8UiSeH2xhfJrSbUr/s/NwtZr1A5O8eLe1w4Pmm4/report/rideshare-economics-101). Teleport is the first mobile app for the TRIP.

For riders, using the Teleport is just like using other ridesharing apps. You pick where you want to go, pay with Apple Pay (or any credit card) and then you're on your way.

![image](https://github.com/TeleportXYZ/TRIP-Guides/assets/19196065/fb009f35-40e8-4b04-bab4-bc896cdfb456)

[Under the hood](https://app.gitbook.com/o/0WSd8UiSeH2xhfJrSbUr/s/NwtZr1A5O8eLe1w4Pmm4/litepaper#id-4.-protocol-design), the Teleport app connects with local rideshare operators and to the TRIP protocol to fulfill your ride. Each rider has a Solana wallet that's spun up when they create an account and all their TRIP Rewards and referrals are tied to this identity.
![image](https://github.com/TeleportXYZ/TRIP-Guides/assets/19196065/a3dc7080-ab56-4ce8-9484-627baf726005)

Similar to how most of don't need to know the inner workings of the internet to use it each day, all of this is abstracted away from riders. You can learn more in the [Litepaper](https://app.gitbook.com/o/0WSd8UiSeH2xhfJrSbUr/s/NwtZr1A5O8eLe1w4Pmm4/litepaper) and the [Rideshare Economics 101](https://app.gitbook.com/o/0WSd8UiSeH2xhfJrSbUr/s/NwtZr1A5O8eLe1w4Pmm4/report/rideshare-economics-101)

**Comment by abhilash710** [comment]
Updated links to point to public pages in guides.trip.dev

**Comment by hwhelchel** [comment]
Abhi this is shaping up to be great.

In my opinion, I'd write it with an angle that places the emphasis on TRIP e.g.

> Teleport is the first mobile app for the TRIP.

You can get rides on the TRIP protocol by using the Teleport mobile app available on iOS.

> Apple Pay (or any credit card)

I'd emphasize whichever payment method most users will statistically use.

> local rideshare operators and to the TRIP protocol to fulfill your ride.

As much as possible remove conjunctions and simplify to make it easier on the reader to follow.

> spun up

Avoid using slang phrases and terms when there is a simpler straight forward alternative. Will make the content more accessible especially to an international or non-tech audience.

> TRIP Rewards and referrals

Simplify and remove conjunction if possible.

> this identity.

Be mindful of introducing new concepts without defining them. If you can reuse a word you already used or something that doesn't need definition, do so.

> Similar to how most of don't need to know the inner workings of the internet to use it each day, all of this is abstracted away from riders.

Don't change the grammatical person in the middle of the doc. I recommend we use second person across the guides as we are writing directly to the reader and we can assume they are in the role of the guide they are currently reading in this case the rider.

None of these are blockers. Happy to have more back and forth discussion here if you feel it's merited. Or feel free to make these suggested edits. All in all, looks great to me to create a change request in GitBook if you haven't yet.

P.S. I'm getting a ton of value out using this tool to help me level up my writing: https://hemingwayapp.com/

**Comment by hwhelchel** [comment]
Merging duplicate here: https://github.com/TeleportXYZ/TRIP-Guides/issues/1

**Comment by Anton-Igudesman** [comment]
Hello I just came across a Teleport post on LinkedIn and would love to get involved in some  open source if possibly - are any of these first issues still being worked on?

**Comment by hwhelchel** [comment]
@Anton-Igudesman yes would be glad to have your contributions. Tag me in an issue you want to help with and explain a bit your approach and we'll go from there.
